src/web/traceViewer/ui/workbench.tsx

```diff
@@ -63,50 +73,88 @@ export const Workbench: React.FunctionComponent<{
   const consoleCount = errors + warnings;
   const networkCount = selectedAction ? modelUtil.resourcesForAction(selectedAction).length : 0;
 
+  const leftTabs: TabbedPaneTab[] = [{
+    id: 'actions',
+    title: 'Actions',
+    count: 0,
+    render: () => <ActionList
+      actions={actions}
+      selectedAction={selectedAction}
+      highlightedAction={highlightedAction}
+      onSelected={action => {
+        setSelectedAction(action);
+      }}
+      onHighlighted={action => setHighlightedAction(action)}
+      setSelectedTab={setSelectedRightTab}
+    />
+  }];
+  if (report.report) {
+    leftTabs.push({
+      id: 'output',
+      title: 'Output',
+      count: 0,
+      render: () => <div className='vbox' style={{ paddingTop: '6px', whiteSpace: 'pre', fontFamily: 'var(--monospace-font)' }}>
+        { (traceSelection ? traceSelection.output : []).map((item, index) => <div
+          key={index} style={{ padding: '0 0 2px 6px' }}>
+          {'text' in item ? highlightANSIText(item.text) : ''}
+          {'buffer' in item ? highlightANSIText(btoa(item.buffer)) : ''}
```

Comment:
I experienced in the past issues with the btoa and unicode symbols - but probably not in our case since both is generated by atob/btoa and they like each-other.